config/currency_iso.json

```diff
@@ -1584,6 +1584,7 @@
     "iso_code": "NIO",
     "name": "Nicaraguan Córdoba",
     "symbol": "C$",
+    "disambiguate_symbol": "NIO$",
```

Comment:
Symbol `C$` is same for CAD and NIO. That's why I added `NIO$` like how there is `CAD$`. This wasn't an issue in this code before as `symbol_before` was set to false, which was wrong and got fixed in earlier commit.